src/test/java/com/rocketden/main/api/ProblemTests.java

```diff
@@ -283,6 +285,34 @@ public void createEditDeleteProblemSuccess() throws Exception {
         assertEquals(ERROR.getResponse(), response);
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {"", "rocket rocket", "12", "$Hello", "jimmy=neutron"})
```

Comment:
For each of these different places you use the parameterized test, could you use different strings? For example instead of the same 5 strings you could cover like 20 or so different examples to minimize the chances of any edge cases we didn't think about (also testing like the reserved keywords and all)

Reply:
Sure, that's a good idea! Done ✅